data_policy.md

```diff
@@ -49,12 +50,14 @@
 
 3.2 Curation of raw data and associated metadata
 
-3.2.1 All raw data will be created in well-defined formats, for which the means of reading the data will be made available by the Facility.
+3.2.1 All raw data will be created in standardized formats, for which the means of reading the data will be made available by the Facility.
 
 3.2.2 Metadata associated with raw data that is automatically captured by instruments will be curated either within the raw data files, within an associated on-line catalogue, or within both.
 
 3.2.3 Raw data and metadata will be stored initially on instrument-related computers, and will be migrated or copied to archival facilities for long-term curation.
 
+3.2.4 After the long-term period, ISIS does not guarantee that raw data will continue to be stored. It is the responsibility of the user to provide storage if they wish to have guaranteed access to raw data after this time. Associated metadata will be kept indefinitely, even if the raw data is deleted.
```

Comment:
Think we need to be careful about wording on this to not cause any scares, as we don't really plan to delete anything. Not sure if this is too blunt.

Reply:
Yes I agree we need to think before we add wording to this effect. Historically ISIS has kept all raw data, to my understanding, but wonder if with IMAT imaging this may need changing. Also, if we are not planning to keep all raw data indefinitely this need to be communicated up front with the users for whom this relates, and something which may not have be considered this yet. So @WHTaylor I think this could instead become an independent issue we can follow up on, and use as a discussion point in a future meeting with others